scss/components/_ckeditor.scss

```diff
@@ -0,0 +1,9 @@
+.ckeditor-toolbar-configuration {
```

Comment:
Because this selector has no companion template in the theme, this technically does not belong to a component.

This set of rules belongs in the `_drupal-overrides.scss` file. Make sure to read the comments in that file and follow procedures therein.